public/app.js

```diff
@@ -48,7 +49,7 @@ class Controller {
         // Update and show the Graphs
         let unsubscribeFunc = DataController.watchProjectTrafficData(
             project,
-            GraphController.updateGraphs
+            GC.updateGraphs
```

Comment:
I still prefer the full name GraphController unless the abbreviation is more readable or common as it well explains its purpose to a new engineer. But will leave it to you to decide

Reply:
The issue is on graph.js class, i.e check line 110 - 119